src/components/App.js

```diff
@@ -35,12 +36,29 @@ class App extends Component {
     return (
       <div className="App">
         <Header/>
-        <Route path="/map">
-          <MapView
-            truckList={this.state.truckList}
-            center={{lat: this.state.lat, lng: this.state.lng}}
-          />
-        </Route>
+        <Switch>
+          <Route exact path='/'>
+            <Redirect to='/login'/>
+          </Route>
+          <Route exact path='/login'>
+            <Form />
+          </Route>
+          <Route exact path="/newuser">
+            <Form/>
+          </Route>
+          <Route exact path="/newlocation">
+            <Form/>
+          </Route>
+          <Route exact path="/trucklist">
```

Comment:
Do we need exact path for all of these?

Reply:
I'm not sure. Do we not need exact path for all of these?

Reply:
haha, well I think its fine either way, but something to look into on best practices maybe. Let's keep it the way it is